engine/launcher/args/args.go

```diff
@@ -87,6 +92,7 @@ func NewEngineServerArgs(
 		DidUserAcceptSendingMetrics: didUserAcceptSendingMetrics,
 		KurtosisBackendType:         kurtosisBackendType,
 		KurtosisBackendConfig:       kurtosisBackendConfig,
```

Comment:
Should we rename this to something more explicit I can't tell how KurtosisBackendConfig plays with KurtosisRemoteBackendConfig

Reply:
Can rename it to `KurtosisLocalBackendConfig` and stick a comment above explaining how the two relates to each other 👍